.eslintrc.json

```diff
@@ -16,14 +16,14 @@
             "error",
             "unix"
         ],
-        "quotes": [
```

Comment:
**Code Quality - NitPick**

Replaced all the double quotes to single quotes. This is to have a standard way of doing things. This is because in our simulation project we use single quotes.